ethstaker_deposit/cli/generate_keys.py

```diff
@@ -163,21 +159,49 @@ def generate_keys_arguments_decorator(function: Callable[..., Any]) -> Callable[
     return function
 
 
+def get_amount_prompt_from_template() -> str:
+    ctx = click.get_current_context(silent=True)
+    chain = ctx.params.get('chain', 'mainnet') if ctx is not None else 'mainnet'
+    devnet_chain_setting = ctx.params.get('devnet_chain_setting')
```

Comment:
Use None as the default if `'devnet_chain_setting'` is not found as a key in this dict.